### PR TITLE
DOC: cleanup about sections

### PR DESCRIPTION
**Pull request by bsipocz**
This should tick the remaining boxes for https://github.com/Caltech-IPAC/irsa-tutorials/issues/12

### CODE REVIEW

**Comment by bsipocz** [comment]
I manually cancelled the tests as forgot to add the label in time when I opened the PR

File: tutorials/parquet-catalog-demos/neowise-source-table-lightcurves.md [hunk 1]
```diff
@@ -455,8 +455,12 @@ You will need to copy the functions and imports from this notebook into a separa
...
 
-About this notebook:
+***
 
-- Author: Troy Raen (Applications Developer, IRSA) and the IPAC Science Platform team
-- Contact: [https://irsa.ipac.caltech.edu/docs/help_desk.html](https://irsa.ipac.caltech.edu/docs/help_desk.html)
-- Updated: 2024-08-08
+## About this notebook
+
+**Author:** Troy Raen (Applications Developer, IRSA) and the IPAC Science Platform team
```
Comment (troyraen) [inline tutorials/parquet-catalog-demos/neowise-source-table-lightcurves.md:462]:
```suggestion
**Author:** Troy Raen (IRSA Developer) and the IPAC Science Platform team
```

File: tutorials/parquet-catalog-demos/neowise-source-table-strategies.md [hunk 2]
```diff
@@ -553,10 +553,12 @@ per_part.loc[per_part.numrows == per_part.numrows.max()]
...
------
+***
 
-About this notebook:
+## About this notebook
 
-- Author: Troy Raen (Applications Developer, IRSA) and the IPAC Science Platform team
-- Contact: [https://irsa.ipac.caltech.edu/docs/help_desk.html](https://irsa.ipac.caltech.edu/docs/help_desk.html)
-- Updated: 2024-08-08
+**Author:** Troy Raen (Applications Developer, IRSA) and the IPAC Science Platform team
```
Comment (troyraen) [inline tutorials/parquet-catalog-demos/neowise-source-table-strategies.md:560]:
```suggestion
**Author:** Troy Raen (IRSA Developer) and the IPAC Science Platform team
```

**Review by troyraen — APPROVED** [review]
LGTM, thanks for doing it. Two suggestions below just to make my job title consistent.

**Comment by bsipocz** [comment]
Thanks Troy. I indeed didn't dive into consistency in job titles, and was also on the fence if to leave in Lauren's email address, as she is not at IRSA. I think we should remove it, but for now I left it unchanged.

**Comment by bsipocz** [comment]
CI has errored out on a ConnectionError with the cosmosim notebook, it's unrelated and we see it from time to time. So I go ahead and merge this now.

**Pull request merged**